switch-configuration/Makefile

```diff
@@ -6,20 +6,37 @@ ifndef PERL
   $(error "perl is not available please install it")
 endif
 
-lint:
-	find . -iname "*.pl" | xargs -i perl -Mstrict -Mdiagnostics -M'warnings FATAL=>q(all)' -cw {}
+all: switch-maps-bundle
 
-secrets:
+lint:	.lint
+
+.lint: config/scripts/*.pl
+	find . -name '*.pl' | xargs '-I{}' perl -Mstrict -Mdiagnostics -M'warnings FATAL=>q(all)' -cw {}
+	touch .lint
```

Comment:
Nice approach to making the makefile dependencies be honored and trigger only when changes to the target actually have updates :+1:

Reply:
Glad you like it.